shub/config.py

```diff
@@ -131,6 +133,8 @@ def save(self, path=None):
             yml['apikeys'] = self.apikeys
             yml['version'] = self.version
             yml['stacks'] = self.stacks
+            if self.eggs:
+                yml['requirements'] = {'eggs': self.eggs}
```

Comment:
I was trying to avoid that discussion, but yes the plan should be to deprecate requirement_file, just not sure we want to do that now.